connect/kafka.mdx

```diff
@@ -7,10 +7,10 @@ If looking to try out without an already setup Kafka endpoint, [see Redpanda qui
 To get the above redpanda container to work with PeerDB on OSS, you can first add it to the PeerDB containers' network:
 ```bash
 # container id of redpanda-0
-docker network connect peerdb_default <redpanda_server_container_id>
+docker network connect peerdb_default $(docker container ls -qf "name=redpanda-0")
 
 # container id of redpanda-console
-docker network connect peerdb_default <redpanda_console_container_id>
+docker network connect peerdb_default $(docker container ls -qf "name=redpanda-console")
```

Comment:
this wasn't working for sai for some reason there was no redpanda-console

Reply:
@serprex  I will pair with @saisrirampur  to see what's going on. This should work.